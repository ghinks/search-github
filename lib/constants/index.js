'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});
const constants = {
  url: 'https://api.github.com/graphql',
  pageSz: 10
};
exports.default = constants;