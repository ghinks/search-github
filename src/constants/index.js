const constants = {
  url: 'https://api.github.com/graphql',
  pageSz: 10
}
export default constants
