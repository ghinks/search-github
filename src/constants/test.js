import constants from './index'
describe('constants', () => {
  it('get the constants', () => {
    expect(constants).toMatchSnapshot()
  })
})
