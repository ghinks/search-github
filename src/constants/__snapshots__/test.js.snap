// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants get the constants 1`] = `
Object {
  "pageSz": 10,
  "url": "https://api.github.com/graphql",
}
`;
